.snippets/code/tutorials/interoperability/replay-and-dry-run-xcms/dry-run-call.ts

```diff
@@ -0,0 +1,32 @@
+import { Binary, createClient, Enum } from "polkadot-api";
+import { withPolkadotSdkCompat } from "polkadot-api/polkadot-sdk-compat";
+import { getWsProvider } from "polkadot-api/ws-provider/web";
+import { assetHub } from "@polkadot-api/descriptors";
+import { sr25519CreateDerive } from "@polkadot-labs/hdkd";
+import { DEV_PHRASE, entropyToMiniSecret, mnemonicToEntropy, ss58Address } from "@polkadot-labs/hdkd-helpers";
```

Comment:
I gotta look into what's happening with the pre-commit hook, but this file hasn't been formatted per our standards - the double quotes are sticking out to me, should be single quotes